src/app/components/publiclists/teamlist/teamlist.component.html

```diff
@@ -9,12 +9,12 @@
             <app-teamcard *ngFor="let team of filteredTeams"
                           [team]="team" [selected]="team['isSelected']" [isOnChallengePage]="isOnChallengePage"
                           (deleteTeamCard)="deleteTeamWrapper()($event)" (editTeamCard)="editTeamWrapper()($event)" (addMembersTeamCard)="addMembersToTeamWrapper()($event)"
-            (selectTeamCard)="selectTeamWrapper()($event)">
+            (selectTeamCard)="selectTeamWrapper()($event)" (DeselectTeamWrapper)="DeselectTeamWrapper()($event)">
```

Comment:
Here you need to change to `deselectTeamWrapper` in order to get [this](https://github.com/Cloud-CV/EvalAI-ngx/pull/289/files#diff-39040dd12941e102fae3dbad153f9262R44) working.

Also change the variable name to `deselectTeamCard` similar to `selectTeamCard`